identifiers/country-pe/national_level.csv

```diff
@@ -0,0 +1,28 @@
+id,name
+ocd-division/country:pe,Peru
+ocd-division/country:pe/province:lma,Municipalidad Metropolitana de Lima
```

Comment:
`region:lim` is the province. This is the municipality.

Reply:
Sorry I am confused. According to https://www.iso.org/obp/ui/#iso:code:3166:PE, region:lim is the region.

However, for lma : it is a province according to Wikipedia.

So, I am not sure if I got your comment.

Reply:
Please link and quote the evidence that "Municipalidad Metropolitana de Lima" (LMA) is a province.

Reply:
`Lima Province is administered by the Metropolitan Municipality of Lima (Municipalidad Metropolitana de Lima), which also administers the city of Lima. Its current mayor is Jorge Muñoz Wells`

Link : https://en.wikipedia.org/wiki/Lima_Province (Government section).

`Peru's territory, according to the Regionalization Law, which was passed on November 18, 2002, is divided into 26 units: 25 regions (regiones; singular: región) and the Lima Province. The regions are subdivided into provinces (provincias), which are composed of districts (distritos). `

Link : https://en.wikipedia.org/wiki/Administrative_divisions_of_Peru (Current division).


However, I agree that it is confusing since the municipality type is used here : https://www.iso.org/obp/ui/#iso:code:3166:PE.

Reply:
Aha, so `ocd-division/country:pe/province:lma` should just have a name of `Lima`, since that ID is the Province of Lima, not the Metropolitan Municipality of Lima.

Reply:
With respect to why Lima province isn't consideered part of the Lima region/department:

> Lima Province, which contains the city of Lima, the country's capital, is located west of the Department of Lima; this province is autonomous and not under the jurisdiction of the Regional Government.

https://en.wikipedia.org/wiki/Department_of_Lima

Reply:
Done. Lima province and Lima region have the same name. Is that Ok ? (I mean not confusing)

Reply:
Yes, it's okay. Names are scoped by their parent division and their type to disambiguate. It's only a problem if both of those are the same. In applications, I've sometimes put the type (e.g. province) in parentheses after the name. This is much simpler to code, compared to removing whatever extra information was added to the name, since that extra information is unpredictable.

Reply:
Got it. Thanks :)